xikolo-tvos/ViewControllers/CourseListViewController.swift

```diff
@@ -19,19 +19,6 @@ class CourseListViewController : AbstractCourseListViewController {
         resultsControllerDelegateImplementation.headerReuseIdentifier = "CourseHeaderView"
     }
 
-    override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
```

Comment:
Did you test this?? I'm pretty sure the app will crash if you don't set `course` on the `CourseTabBarController`.